integration/agent/agent_test.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This will run a containers and then check
+# whether the kata-agent threads namespaces
+# were changed during create container processes.
+
+set -e
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../metrics/lib/common.bash"
+
+CTR_RUNTIME="${CTR_RUNTIME:-io.containerd.kata.v2}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+IMAGE="${IMAGE:-docker.io/library/busybox:latest}"
+
+function setup() {
```

Comment:
Nit: `function` keyword is redundant.

Reply:
Thanks @jodh-intel, all comments were addressed.